assets/texts/en/permissions.md

```diff
@@ -17,8 +17,9 @@ These permissions are requested when doing some specific actions, if they are ne
 ## Hidden permissions
 Additionally, it requests these permissions, which are not requested in Firefox when the add-on is installed, as they are not major permissions.
 
-| Internal Id | Permission                   | Explanation                                                       |
-|:------------|:-----------------------------|:------------------------------------------------------------------|
-| `activeTab` | Access current tab website   | Needed for getting the URL of the current website for the QR code |
-| `storage`   | Access local storage         | Needed for saving options                                         |
-| `menus`     | Modify browser context menus | Needed for adding context menus "QR code from selection" (etc.)   |
+| Internal Id    | Permission                   | Explanation                                                       |
+|:---------------|:-----------------------------|:------------------------------------------------------------------|
+| `activeTab`    | Access current tab website   | Needed for getting the URL of the current website for the QR code |
+| `storage`      | Access local storage         | Needed for saving options                                         |
+| `menus`        | Modify browser context menus | Needed for adding context menus "QR code from selection" (etc.)   |
+| `clipboardRead`| Access clipboard content     | Needed for using clipboard content to create QR code              |
```

Comment:
Thanks for doing the doc part, too! However, currently it is in the wrong section, as currently it would be an "Installation permissions" and actually _not_ be hidden.

_However_, before you change that, see my suggestion below. Actually I'd like if it were an optional permission, so this would be placed under "Feature-specific (optional) permissions".

Reply:
See https://extensionworkshop.com/documentation/develop/request-the-right-permissions/#request-permissions-at-runtime